cpu/atmega1284p/include/atmega_pcint.h

```diff
@@ -0,0 +1,7 @@
+#ifndef ATMEGA_PCINT_H
+#define ATMEGA_PCINT_H
+#define ATMEGA_PCINT_MAP_PCINT0 { GPIO_PIN(0, 0), GPIO_PIN(0, 1), GPIO_PIN(0, 2), GPIO_PIN(0, 3), GPIO_PIN(0, 4), GPIO_PIN(0, 5), GPIO_PIN(0, 6), GPIO_PIN(0, 7) }
```

Comment:
Please use `GPIO_PIN(PORT_A, 0)` instead of `GPIO_PIN(0, 0)`. This will make reading it easier, as the code better matches the names used in the datasheet. (E.g. the pins are name PB0, PA7, ...)

Reply:
absolutely true :)